LocationManager/INTULocationManager/INTULocationManager.h

```diff
@@ -125,6 +130,33 @@ __INTU_ASSUME_NONNULL_BEGIN
 /** Immediately cancels the location request (or subscription) with the given requestID (if it exists), without executing the original request block. */
 - (void)cancelLocationRequest:(INTULocationRequestID)requestID;
 
+#pragma mark Heading Requests
+
+/** 
+ Whether or not the CLLocationManager is currently sending heading updates. 
+ */
+@property (nonatomic, assign, readonly) BOOL isUpdatingHeading;
+
+/**
+ * Specifies the minimum amount of change in degrees needed for a heading service update. Observers will not be notified of updates less than the stated filter value.
+ *
+ * @discussion The default value for this is 1.
+ */
+@property (nonatomic) INTUHeadingFilterAccuracy headingFilter;
```

Comment:
What are your thoughts about exposing an API like:

```objc
- (void)applyDistanceFilter:(CLLocationDistance)distanceFiter toLocationRequest:(INTULocationRequestID)requestID;
```

This would let you apply a distance filter to a specific location request, after the request has been created. Just an idea :)